app/Jobs/FirmwareDownloadJob.php

```
@@ -40,8 +41,8 @@ public function handle(): void
             ]);
         } catch (ConnectionException $e) {
             Log::error('Firmware download failed: '.$e->getMessage());
-        } catch (\Exception $e) {
+        } catch (Exception $e) {
             Log::error('An unexpected error occurred: '.$e->getMessage());
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (Clean Code Rules)

(StaticAccess)